l3packages/xtemplate/xtemplate-2023-10-10.sty

```diff
@@ -0,0 +1,1373 @@
+%%
+%% This is file `xtemplate.sty',
```

Comment:
I think such files should identify themselves as a frozen version, e.g., for array-2016-... there is

%%
%% This is an old release of `array.sty',
%%

at the top